### PR TITLE
feat: npm install express and create index.js

### PR DESCRIPTION
**Pull request by DonLe27**
noice

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "eslint": "^7.6.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-react": "^7.20.5",
+    "express": "^4.17.1",
```
Comment (BryanPan342) [inline package.json]: let's delete this file

**Review by BryanPan342 — DISMISSED** [review]
can we remove the root package-lock.json and the express entry in package.json?

**Review by josephtclegg — APPROVED** [review]

**Comment by BryanPan342** [comment]
@mergifyio refresh

**Review by josephtclegg — APPROVED** [review]

**Review by BryanPan342 — APPROVED** [review]
LGTM

**Review by josephtclegg — APPROVED** [review]

**Pull request merged**